src/stories/containers/TransparencyReport/components/HeaderWithIcon/TooltipHeader.tsx

```diff
@@ -0,0 +1,104 @@
+import styled from '@emotion/styled';
+import { CustomLink } from '@ses/components/CustomLink/CustomLink';
+import { useThemeContext } from '@ses/core/context/ThemeContext';
+import React from 'react';
+import type { WithIsLight } from '@ses/core/utils/typesHelpers';
+import type { CSSProperties } from 'react';
+
+interface Props {
+  description: string;
+  mipNumber: string;
+  link: string;
+  name: string;
+  style?: CSSProperties;
+}
+
+const HeaderToolTip: React.FC<Props> = ({ description, mipNumber, link, style, name }) => {
+  const { isLight } = useThemeContext();
+  return (
+    <Container style={style}>
+      <Description isLight={isLight}>{description}</Description>
+      <Source isLight={isLight}>Source</Source>
+      <ContainerLinkWithMip isLight={isLight}>
+        <MipNumber>{mipNumber}</MipNumber>
+        <ContainerLink>
+          <CustomLink
+            children={name}
+            withArrow
+            marginLeft="7px"
+            href={link}
+            iconWidth={10}
+            fontWeight={400}
+            iconHeight={10}
+            style={{
+              lineHeight: '15px',
+              fontSize: '12px',
+              letterSpacing: '0px',
+              marginLeft: 0,
+              paddingRight: 0,
+              whiteSpace: 'pre-line',
+              overflowWrap: 'break-word',
+              width: '259px',
+            }}
```

Comment:
would be nice to extend this component using `styled(CustomLink)...` to avoid adding inline styles

Reply:
done